emails/templates/partials/alerting/instance_details.mjml

```diff
@@ -76,80 +52,56 @@
 <mj-section padding="0 25px" text-align="left">
   <mj-column padding="10px" background-color="#111217" border="1px solid #2f3037">
     <mj-text>
-      <mj-raw>
-        {{ range $refID, $value := .Values }}
-      </mj-raw>
+      <mj-raw>{{ range $refID, $value := .Values }}</mj-raw>
       {{ $refID }}={{ $value }}&nbsp;
-      <mj-raw>
-        {{ end }}
-      </mj-raw>
+      <mj-raw>{{ end }}</mj-raw>
     </mj-text>
   </mj-column>
 </mj-section>
-<mj-raw>
-  {{ end }}
-</mj-raw>
+<mj-raw>{{ end }}</mj-raw>
 
 <!-- Labels and annotations -->
 <mj-section>
   <mj-column>
     <!-- Labels -->
-    <mj-raw>
-      {{ if .Labels.SortedPairs }}
-    </mj-raw>
+    <mj-raw>{{ if .Labels.SortedPairs }}</mj-raw>
     <mj-text>
       <strong>Labels</strong>
     </mj-text>
 
     <mj-table color="white">
-      <mj-raw>
-        {{ range .Labels.SortedPairs }}
-      </mj-raw>
+      <mj-raw>{{ range .Labels.SortedPairs }}</mj-raw>
       <tr>
         <td>
           <strong>{{ .Name }}</strong>
         </td>
-        <td>
-          {{ .Value }}
-        </td>
+        <td>{{ .Value }}</td>
       </tr>
-      <mj-raw>
-        {{ end }}
-      </mj-raw>
+      <mj-raw>{{ end }}</mj-raw>
     </mj-table>
 
-    <mj-raw>
-      {{ end }}
-    </mj-raw>
+    <mj-raw>{{ end }}</mj-raw>
 
-    <!-- Annotations -->
-    <mj-raw>
-      {{ if .Annotations.SortedPairs }}
-    </mj-raw>
+    <!-- Annotations, we exclude "description" and "summary" because those have their own dedicated section -->
+    <mj-raw>{{ if (without .Annotations.SortedPairs.Names "description" "summary") }}</mj-raw>
```

Comment:
For some reason `mjml` decided to re-format everything but the important changes are here